smartseq2_single_sample/run_multi_samples/scripts/README.md

```diff
@@ -0,0 +1,39 @@
+# Running Time and Cost Calculation
+This script is original post in [dsde-pipeline](https://github.com/broadinstitute/dsde-pipelines/blob/develop/scripts/calculate_cost.py), which is used to calculate running time and cost of workflow in google cloud. 
+We modified this python script in order to calculate cost of a subworkflow.
```

Comment:
I'd like to avoid having multiple copies of this script in different repos. We should figure out the one repo where this should live and make all improvements to that one copy.